src/panel/Menu.jsx

```diff
@@ -36,58 +37,47 @@ const Menu = () => {
   };
 
   return <Fragment>
-    <MenuButton onClick={open} />
+    <button
+      type="button"
+      className={cx('menu__button', { 'menu__button--active': isOpen })}
+      onClick={toggle}
+      title={_('Menu')}
+    >
+      <IconMenu />
+    </button>
 
     {isOpen && ReactDOM.createPortal(<div className="menu">
       <div className="menu__overlay" onClick={close} />
 
       <div className="menu__panel">
-        <div className="menu__panel__top">
-          <h2 className="menu__panel__top__title">
-            <i className="menu__panel__top__icon icon-map" />
-            <span>Qwant Maps</span>
-            <i className="icon-x menu__panel__top__close" onClick={close} />
-          </h2>
-        </div>
-
-        <div className="menu__panel__items_container">
-          <div className="menu__panel__section menu__panel__section-internal">
-            <button className="menu__panel__action"
-              onClick={() => { navTo('/', { focusSearch: true }); }}
-            >
-              <img
-                className="menu__panel__action__icon"
-                src="./statics/images/magnifier-dark.svg" alt=""
-              />
-              <span>{_('Search', 'menu')}</span>
-            </button>
-            {isDirectionActive &&
-              <button className="menu__panel__action" onClick={() => {
-                Telemetry.add(Telemetry.MENU_ITINERARY);
-                navTo('/routes/');
-              }}>
-                <i className="menu__panel__action__icon icon-corner-up-right" />
-                <span>{_('Directions', 'menu')}</span>
-              </button>
-            }
-            <button className="menu__panel__action" onClick={() => {
+        <Flex className="menu-top u-mb-l">
+          <CloseButton circle onClick={close} />
+        </Flex>
+        <div className="menu-items">
+          <MenuItem
+            onClick={e => {
+              e.preventDefault();
               Telemetry.add(Telemetry.MENU_FAVORITE);
               navTo('/favs/');
-            }}>
-              <i className="menu__panel__action__icon icon-icon_star" />
-              <span>{_('Favorites', 'menu')}</span>
-            </button>
-            <a className="menu__panel__action menu__panel__section_title__link"
-              href="https://github.com/QwantResearch/qwantmaps/blob/master/contributing.md"
-              rel="noopener noreferrer"
-              target="_blank"
-            >
-              <i className="menu__panel__action__icon icon-zap" />
-              <span>{_('How to contribute', 'menu')}</span>
-            </a>
-          </div>
-
-          {menuItems.map(menuItem => <MenuItem key={menuItem.sectionName} menuItem={menuItem} />)}
+            }}
+            icon={<Heart width={16} color={PINK_DARK} />}
+          >
+            {_('Favorites', 'menu')}
```

Comment:
The mockup specified "My favorites", which is consistent with the title used on the FavoritesPanel header.

Reply:
Fixed.